public/js/pages/editor.js

```diff
@@ -3,7 +3,7 @@ define(['jquery', 'localized', 'nunjucks', 'base/ui', 'moment', 'uri', 'makeapi'
     'use strict';
 
     var MAKE_VIEW = 'make-templates/make-admin-search.html',
-      MAKE_URL = $('body').data('endpoint'),
+      MAKE_URL = $('meta[name='make-endpoint']').attr('content'),
```

Comment:
Incorrect single quote use.

Either escape the inner quotes or make them double quotes.